cmd/if-mover/main.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/vishvananda/netlink"
+	"github.com/vishvananda/netns"
+)
+
+const (
+	ifBase     = "mlink"
+	hostNsFile = "/tmp/hostns.txt"
+	multusLink = "net1"
+)
+
+func main() {
+
+	log.Println("Determining host namespace id")
```

Comment:
Logging in rook is based on `capnslog` instead of `log`. For example usage, search for `logger.Info`